mr/src/test/java/org/apache/iceberg/mr/mapreduce/TestIcebergInputFormat.java

```diff
@@ -520,12 +554,14 @@ private DataFile writeFile(
 
     DataFiles.Builder builder = DataFiles.builder(table.spec())
         .withPath(file.toString())
-        .withFormat(format)
+        .withFormat(this.fileFormat)
         .withFileSizeInBytes(file.length())
         .withMetrics(appender.metrics());
+
     if (partitionData != null) {
       builder.withPartition(partitionData);
     }
+
```

Comment:
Let's minimize these non-functional changes that usually get called out in Iceberg PRs.